styx-cli/src/main/java/com/spotify/styx/cli/Main.java

```diff
@@ -363,7 +365,9 @@ private String apiUrl(CharSequence... parts) {
   }
 
   private ByteString send(Request request) throws ExecutionException, InterruptedException {
-    final Response<ByteString> response = client.send(request).toCompletableFuture().get();
+    final Response<ByteString> response =
+        client.send(request.withHeader("Client-Version", STYX_CLI_VERSION)).toCompletableFuture()
```

Comment:
Should we include "Styx" in the header? E.g. `Styx-Client-Version: Styx CLI 1.2.3`. 

Alternatively, would it perhaps make sense to use the `User-Agent` header? E.g. `User-Agent: Styx CLI 1.2.3`.